app/assets/stylesheets/dlme.scss

```diff
@@ -253,28 +262,34 @@ body {
 }
 
 .topbar {
+  position: absolute;
+  width: 100vw;
```

Comment:
Why is these necessary? Topbar is already full width.

Reply:
The existing styles don't have an effect once its position is changed to `absolute`, which is necessary to overlay it on the masthead.

Reply:
Why are we changing to absolute? I'm really struggling to understand that purpose of these changes.

Reply:
We overrode several spotlight templates in order to construct an HTML setup where the navbar was inside the masthead, in order to implement the current design. This PR gets rid of all the overrides in favor of creating that appearance via a small amount of CSS instead, so we don't diverge from Spotlight so much. It also means that we get to keep the default navbar, so it automatically appears on pages that do not have a masthead, like login pages, which is part of the ticket.

Reply:
Specifically, we use absolute so that the navbar can be positioned over the top of the masthead, which is its following sibling.